virttest/utils_v2v.py

```diff
@@ -455,6 +455,9 @@ def __init__(self, test, params, env):
             raise ValueError("Doesn't support %s target now" % self.target)
 
     def create_session(self, timeout=480):
+        if self.session:
```

Comment:
Are we considering adding another parameter to have options on this? Like, 'single_session=True'

Reply:
Right now we don't have such use case.  It could be implemented if needed in future.